image-compressor-service-default.properties

```diff
@@ -0,0 +1,7 @@
+#iam image-compressor-service config
+image_compressor_resize_factor_fx=0.25
```

Comment:
Can we not name it as mosip.bio.image.compressor.resize.factor.fx? 
Similarly for other parameters as well